src/components/index/BlogSection.tsx

```diff
@@ -21,45 +22,51 @@ export interface Blog {
 const posts = [
     {
         id: 1,
-        title: 'Boost your conversion rate',
+        title: 'Sunday Celebratrion',
         href: '#',
         description:
             'Illo sint voluptas. Error voluptates culpa eligendi. Hic vel totam vitae illo. Non aliquid explicabo necessitatibus unde. Sed exercitationem placeat consectetur nulla deserunt vel. Iusto corrupti dicta.',
         imageUrl:
-            'https://images.unsplash.com/photo-1496128858413-b36217c2ce36?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=3603&q=80',
+            'https://res.cloudinary.com/dpbuspblx/image/upload/v1679958909/cla-cta/Sunday_Celebration_IG_wfnxzj.png',
         date: 'Mar 16, 2020',
         datetime: '2020-03-16',
         author: {
             name: 'Michael Foster',
             imageUrl:
                 'https://images.unsplash.com/photo-1519244703995-f4e0f30006d5?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=facearea&facepad=2&w=256&h=256&q=80',
         },
+        imageWidth:1080,
+        imageHeight:1080
     },
     {
         id: 2,
-        title: 'Improve Your Productivity',
+        title: 'Improve Your Productivity and Work',
         href: '#',
         description: 'Dolorum voluptatem ab molestias in magni nostrum. Sint deleniti corporis nam et deserunt illo consequatur. Cupiditate expedita fugit illum. Officia veniam atque dolorem a. Quisquam corrupti eveniet nam. Est fugit doloremque repellendus omnis quis id voluptatum.',
-        imageUrl: 'https://images.unsplash.com/photo-1496128858413-b36217c2ce36?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=3603&q=80',
+        imageUrl: 'https://res.cloudinary.com/dpbuspblx/image/upload/v1679958907/cla-cta/Sermon_Quote_2_Twitter_iotdpu.png',
         date: 'May 12, 2020',
         datetime: '2020-05-12',
         author: {
             name: 'Julia Thomas',
             imageUrl: 'https://images.unsplash.com/photo-1566053618601-6c0b8bb10c15?ixlib=rb-1.2.1&auto=format&fit=facearea&facepad=2&w=256&h=256&q=80',
         },
+        imageWidth:1024,
```

Comment:
we don't need these since we wouldn't get them from the CMS anyhow...

Reply:
cool cool